package.json

```diff
@@ -60,17 +60,11 @@
     "pinkie-promise": "^2.0.0"
   },
   "devDependencies": {
+    "ava": "*",
```

Comment:
Perhaps use a proper semver range (`^X.X.X`)? So the build wouldn't break when breaking changes are released in a major version.